hrms/payroll/doctype/salary_slip/salary_slip.py

```diff
@@ -466,10 +466,11 @@ def get_payment_days(self, joining_date, relieving_date, include_holidays_in_tot
 				return
 
 		end_date = getdate(self.end_date)
+		employee_status = froggy.get_cached_value("Employee", self.employee, "status")
```

Comment:
Move this check after the branch:

`if relieving_date`

We should only fetch this value if relieving date is set.